package.json

```diff
@@ -138,9 +138,6 @@
       "optional": true
     }
   },
-  "resolutions": {
```

Comment:
Can we keep this? During development we use it to ensure only one version of `type-fest` is installed.

Reply:
Oh, I see.